cluster-operator/pom.xml

```diff
@@ -17,7 +17,7 @@
     </licenses>
 
     <properties>
-        <mockserver.version>5.11.1</mockserver.version>
+        <mockserver.version>5.13.2</mockserver.version>
```

Comment:
This is not new ... but why is this in the Cluster Operator pom.xml?

Reply:
I suppose this property could be moved to the parent? It is only used by the cluster operator tests though?

Reply:
Yeah it's used only here but maybe it's better to move to the parent to have all version definitions in one place.